.github/workflows/deploy-staging.yml

```
@@ -0,0 +1,57 @@
+        run: npm run lint
+
+      - name: Run tests
+        run: npm run test
+
+  deploy:
+    name: Deploy to Staging
+    runs-on: ubuntu-latest
+    needs: test
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)